lib/KTable/index.vue

```diff
@@ -0,0 +1,480 @@
+<template>
+
+  <div ref="tableWrapper" class="k-table-wrapper">
+    <table class="k-table" role="grid">
+      <caption v-if="caption">
+        {{ caption }}
+      </caption>
+      <thead>
+        <tr>
+          <th
+            v-for="(header, index) in headers"
+            :key="index"
+            tabindex="0"
+            :aria-sort="sortable && header.dataType !== DATA_TYPE_OTHERS ? getAriaSort(index) : null"
+            :class="{
+              [$computedClass(coreOutlineFocus)]: true,
+              sortable: sortable && header.dataType !== DATA_TYPE_OTHERS,
+              'sticky-header': true,
+              'sticky-column': index === 0,
+            }"
+            :style="[getHeaderStyle(header),
+                     sortKey === index ? { color: $themeBrand.primary.v_1000 } : { color: $themePalette.grey.v_800 },
+                     { backgroundColor: $themePalette.white } ,
+                     focusedColIndex === index ? { backgroundColor: $themePalette.grey.v_50 } : {}]"
+            role="columnheader"
+            :aria-colindex="index + 1"
+            @click="sortable && header.dataType !== DATA_TYPE_OTHERS ? handleSort(index) : null"
+            @keydown="handleKeydown($event, -1, index)"
+          >
+            <!--@slot Scoped slot for customizing the content of each header cell. Receives the following slot props - `header`: The current header object & `index`: The index of the current header.-->
+
+            <slot name="header" :header="header" :index="index">
```

Comment:
```suggestion
            <slot name="header" :header="header" :colIndex="index">
```

Few tweaks for clarity and consistency with the `cell` slot. Note that this would also need to be updated in the documentation examples.

Reply:
done